### PR TITLE
quick linter pass and standardization

### PR DESCRIPTION
**Pull request by faddat**
Before beginning on the data locality work, a quick linter so can always see if there's an issue.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -32,38 +32,37 @@ tools:
...
-		go test $(LDFLAGS) -run=NOTEST -bench=RandomBytes . && \
-		go test $(LDFLAGS) -tags cleveldb,rocksdb,pebbledb -run=NOTEST -bench=Small . && \
-		go test $(LDFLAGS) -tags cleveldb,rocksdb,pebbledb -run=NOTEST -bench=Medium . && \
-		go test $(LDFLAGS) -tags cleveldb,rocksdb,pebbledb -run=NOTEST -timeout=30m -bench=Large . && \
-		go test $(LDFLAGS) -run=NOTEST -bench=Mem . && \
-		go test $(LDFLAGS) -run=NOTEST -timeout=60m -bench=LevelDB .
+		go test $(LDFLAGS) -bench=RandomBytes . && \
+		go test $(LDFLAGS) -tags boltdb,badgerdb,pebbledb -bench=Small . && \
+		go test $(LDFLAGS) -tags boltdb,badgerdb,pebbledb -bench=Medium . && \
+		go test $(LDFLAGS) -tags boltdb,badgerdb,pebbledb -timeout=30m -bench=Large . && \
```
Comment (robert-zaremba) [inline Makefile:63]: so, are we still using boltdb and badger db?

File: Makefile [hunk 2]
```diff
@@ -32,38 +32,37 @@ tools:
...
+		go test $(LDFLAGS) -bench=Mem . && \
+		go test $(LDFLAGS) -timeout=60m -bench=LevelDB .
```
Comment (robert-zaremba) [inline Makefile:65]: Why do you remove `-run=NOTEST` flag? it's needed to not start any test. We don't want it to impact benchmarks.

**Review by robert-zaremba — COMMENTED** [review]
We should keep `-run=NOTEST` flag

File: Makefile (hunk 1 above)
Comment (faddat) [reply Makefile:63]: no

**Comment by faddat** [comment]
in fact the bottom line here is this is just a bad PR.


Sorry!

**Pull request closed without merging**